source/cli/index.md

```diff
@@ -17,5 +17,44 @@ You can find it here with its documentation [http://cli.scalingo.com](http://cli
 
 * You can use tne environment `SCALINGO_APP` instead of using the `--app` flag
 * If your current directory is the base directory of your project
-  and that your git repository has a remote named 'scalingo', your
+  and that your git repository has a remote named 'scalingo', you
   don't need to specify `--app <name>` it will be detected automatically.
+  * If you want to specify the remote name, you can do it by using `--remote` or `-r` flag followed
+    by the name
+
+## Command Completion
+
+### Bash
+
+* Make sure bash completion is installed. If you use a current Linux in a non-minimal installation, bash completion should be available. On a Mac, install with `brew install bash-completion`
+
+* Get bash completion script in the correct folder:
+  * Linux users :
+
+```bash
+curl "https://raw.githubusercontent.com/Scalingo/cli/master/cmd/autocomplete/scripts/scalingo_complete_bash" > /etc/bash_completion.d/scalingo_complete_bash.sh
+```
+  * Mac users :
+
+```bash
+curl "https://raw.githubusercontent.com/Scalingo/cli/master/cmd/autocomplete/scripts/scalingo_complete_bash" > /usr/local/etc/bash_completion.d/scalingo_complete_bash.sh
+```
+
+
+* Reload your shell in order to make the completion available: `exec bash -l`
+
+### Zsh
+
+* Get zsh completion script in the folder you want (~/.zsh/completion/):
```

Comment:
The user should be guided, in this case he may be lost, you need to be more precise

ie. create the directory .zsh/completion etc.